app/models.py

```diff
@@ -511,6 +511,9 @@ def update_from_json(self, data):
         self.data = current_data
         self.updated_at = datetime.utcnow()
 
+    def purge_nulls(self):
+        self.data = dict((k, v) for k, v in iteritems(self.data) if v is not None)
```

Comment:
`for k, v in self.data.items()` would've worked here too since we're not too fussed about laziness on python2